components/dashboard/src/repositories/detail/ConfigurationDetailPrebuilds.tsx

```diff
@@ -4,27 +4,59 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-import { FC } from "react";
+import { FC, useState } from "react";
 import { Configuration } from "@gitpod/public-api/lib/gitpod/v1/configuration_pb";
 import { ConfigurationSettingsField } from "./ConfigurationSettingsField";
 import { Heading3, Subheading } from "@podkit/typography/Headings";
+import { Switch } from "@podkit/switch/Switch";
+import { TextMuted } from "@podkit/typography/TextMuted";
 
 type Props = {
     configuration: Configuration;
 };
 export const ConfigurationDetailPrebuilds: FC<Props> = ({ configuration }) => {
+    // TODO: hook this up to just use configuration as state and wire up optimistic update for mutation
+    const [enabled, setEnabled] = useState(!!configuration.prebuildSettings?.enabled);
+
     return (
         <>
             <ConfigurationSettingsField>
                 <Heading3>Prebuilds</Heading3>
                 <Subheading className="max-w-lg">
                     Prebuilds reduce wait time for new workspaces. Enabling requires permissions to configure repository
                     webhooks.{" "}
```

Comment:
I think this is safe to remove since it's a part of the switch now. Same goes for the link below, but GitHub did not let me expand my suggestion that far

```suggestion
                    Prebuilds reduce wait time for new workspaces.
```